ui/app/routes/vault/cluster/oidc-callback.js

```diff
@@ -6,7 +6,15 @@ export default Route.extend({
     // left blank so we render the template immediately
   },
   afterModel() {
-    let { auth_path: path, code, state } = this.paramsFor(this.routeName);
+    const queryString = decodeURIComponent(window.location.search);
```

Comment:
I'm not sure how this was working before, but when I try testing with this code OIDC login fails and `queryString` is nothing. 

I trust that Arnav's fix did work, though, so my suggestion would be to conditionally add it so this method would look like this: 
```js
  afterModel() {
    let { auth_path: path, code, state } = this.paramsFor(this.routeName);
    let { namespaceQueryParam: namespace } = this.paramsFor('vault.cluster');
    // namespace from state takes precedence over the cluster's ns
    if (state?.includes(',ns=')) {
      [state, namespace] = state.split(',ns=');
    }
    // some SSO providers do not return a url-encoded string, check for namespace using URLSearchParams
    const queryString = decodeURIComponent(window.location.search);
    const urlParams = new URLSearchParams(queryString);
    const checkState = urlParams.get('state');
    if (checkState?.includes(',ns=')) {
      [state, namespace] = checkState.split(',ns=');
    }
    path = window.decodeURIComponent(path);
    const source = 'oidc-callback'; // required by event listener in auth-jwt component
    const queryParams = { source, path: path || '', code: code || '', state: state || '' };
    if (namespace) {
      queryParams.namespace = namespace;
    }
    window.opener.postMessage(queryParams, window.origin);
  }
```